iib/workers/tasks/build.py

```diff
@@ -671,10 +671,19 @@ def inspect_related_images(bundles: List[str], request_id) -> None:
     :raises IIBError: if one of the bundles does not have the pullable related_image.
     """
     invalid_related_images = []
+    replace_related_image_registry = None
     for bundle in bundles:
         manifest_location = get_image_label(
             bundle, "operators.operatorframework.io.bundle.manifests.v1"
         )
+        bundle_registry = bundle.split('/')[0]
+        if (
```

Comment:
This entire blob can be replaced with  

`replace_related_image_registry = worker_config.iib_related_image_registry_replacement.get(bundle_registry)`

Reply:
In fact, it could be a single line like

`replace_related_image_registry = worker_config.iib_related_image_registry_replacement.get(bundle.split('/')[0])`

getting rid of line 679 too, no? I am open to keeping 679 for clarity, though

Reply:
Yeah I created `bundle_registry` only for ease of read

If I directly use `replace_related_image_registry = worker_config.iib_related_image_registry_replacement.get(bundle_registry)` and if iib_related_image_registry_replacement config is not set, it fails with keyError, so we cannot have the config unset, it will always need a default.

Reply:
yeah, and the default can be an empty dict, right?

Reply:
I went with default option of config not set, but I can change it to dict based on thread on config file

Reply:
Updated